nifi-stateless/nifi-stateless-bundle/nifi-stateless-engine/src/main/java/org/apache/nifi/stateless/flow/StandardStatelessDataflowFactory.java

```diff
@@ -83,14 +81,12 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 
 public class StandardStatelessDataflowFactory implements StatelessDataflowFactory<VersionedFlowSnapshot> {
     private static final Logger logger = LoggerFactory.getLogger(StandardStatelessDataflowFactory.class);
-    private static final EncryptionMethod ENCRYPTION_METHOD = EncryptionMethod.MD5_256AES;
+    private static final String PROPERTY_ENCRYPTION_METHOD = "NIFI_PBKDF2_AES_GCM_256";
```

Comment:
Suggest referencing the enum directly rather than a String PropertyEncryptionMethod.NIFI_PBKDF2_AES_GCM_256.toString()
This requires making the enum public. Do you see a problem with that?

Reply:
Thanks @markobean, that makes sense, I will make the change.